### PR TITLE
Add guidelines for adding or changing guidelines and to upgrade a guideline to a policy

### PR DESCRIPTION
**Pull request by sshiells-scottlogic**
Add guidelines for adding or changing guidelines and to upgrade a guideline to a policy

### CODE REVIEW

File: docs/community-guidelines/adding-modifying-guidelines.md
Comment (rgriffiths-scottlogic) [inline docs/community-guidelines/adding-modifying-guidelines.md:1]: guidlines --> guidelines

File: docs/community-guidelines/guidelines-to-policies.md
Comment (rgriffiths-scottlogic) [inline docs/community-guidelines/guidelines-to-policies.md:1]: Another guidline --> guideline

Comment (rgriffiths-scottlogic) [inline docs/community-guidelines/guidelines-to-policies.md:1]: Is it a "SC member sponsor"? As in, the sponsor is a SC member? Or it is a SC Member, sponsor, or the WG Lead. I just want to make sure that we all know whether it's two "roles" or three that we're talking about!

Comment (sshiells-scottlogic) [reply docs/community-guidelines/guidelines-to-policies.md:1]: The sponsor is a SC member.

Comment (rgriffiths-scottlogic) [reply docs/community-guidelines/guidelines-to-policies.md:1]: Ah, I see, later on you remove the word "member" so perhaps that's the trick for this first time round? Then it's clearer that it's only two roles.

Comment (rgriffiths-scottlogic) [inline docs/community-guidelines/guidelines-to-policies.md:1]: and if approved by the majority the --> and, if approved by the majority, the

**Review by rgriffiths-scottlogic — COMMENTED** [review]
I've added a few solitary comments for consideration and, once you've dealt with these, I'd be happy to approve!

File: docs/community-guidelines/guidelines-to-policies.md
Comment (sshiells-scottlogic) [reply docs/community-guidelines/guidelines-to-policies.md:1]: fixed

Comment (sshiells-scottlogic) [reply docs/community-guidelines/guidelines-to-policies.md:1]: Added member back.

**Review by rgriffiths-scottlogic — APPROVED** [review]
Nice work, Stevie! A super Christmas present 🎁

**Review by eddie-knight — APPROVED** [review]

**Pull request merged**